### DISCUSSION TITLE
Is there a way to overwrite the subscription start date?

### DISCUSSION THREAD

**Discussion by timmyd87** [pay-rails/pay#987] [Q&A]
Use case is fixed renewal on the 1st of July every year - so it's a yearly subscription but not pro rated, meaning we need to back date the start date to 1 July and get it to run until 30 June.

Can this be done? What's the best way to approach with Pay?

**Comment by excid3** [comment]
You'll want to consult the Stripe API for this. We just forward along options to their API.
